go/fasthttp/main.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"bytes"
+
+	"github.com/valyala/fasthttp"
+)
+
+var (
+	strGET            = []byte("GET")
+	strPOST           = []byte("POST")
+	strSlash          = []byte("/")
+	strSlashUserSlash = []byte("/user/")
```

Comment:
is there any way to have a dynamic URL ?

I mean `strSlashUserSlash` could lead either to `/user/0` than `/user/$hacking_value*` ?

Reply:
That's why I'm using `bytes.HasPrefix` to check. There is no default router build into fasthttp.

Reply:
ok, it seems different than other frameworks I know :heart: